dll/win32/gdiplus/x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.23038_none_deadbeef.manifest

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
-  <assemblyIdentity type="win32" name="Microsoft.Windows.GdiPlus" version="1.0.100.0" processorArchitecture="x86" />
+  <assemblyIdentity type="win32" name="Microsoft.Windows.GdiPlus" version="1.1.7601.23038" processorArchitecture="" publicKeyToken="6595b64144ccf1df"/>
```

Comment:
sometimes processorArchitecture is empty, sometimes it's `*`, i'm not sure which is best?

Reply:
I am not really sure. I think we should keep x86 and x64 copies.